Formula/x11vnc.rb

```diff
@@ -25,7 +25,7 @@ def install
       "--disable-debug",
       "--disable-dependency-tracking",
       "--prefix=#{prefix}",
-      "--mandir=#{man}"
+      "--mandir=#{man}",
```

Comment:
If you wouldn't mind, the same comment I made IRT using a `%W` style array in the other PR would be great here as well.